lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/EvaluatorRequestorBridge.java

```diff
@@ -85,4 +99,17 @@ public int getEvaluatorNumber() {
   @Override
   public void close() {
   }
+
+  public byte[] getDefinedRuntimes(){
+    LOG.log(Level.INFO, "Defined Runtimes :" + StringUtils.join(this.definedRuntimes, ','));
```

Comment:
Put this into a lower log level.

Nitpick: The assembly of the string parameter here is expensive. So maybe guard it with a check for the log level?